guides/doc-Configuring_DNS_DHCP_TFTP/master.adoc

```diff
@@ -0,0 +1,18 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: admin-network
+:admin-network:
+:ProductName: {ProjectServer}
+
+= {ConfiguringDNSDHCPTFTPDocTitle}
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+include::common/assembly_configuring-dns-integration.adoc[leveloffset=+1]
+
```

Comment:
Just a suggestion. 

You might want to remove "integration" from the assembly names because I don't think there are benefits to making our headers longer than they need to be. From the user's perspective, "Configuring DNS" is the same as "Configuring DNS integration".

Reply:
With "integrating", it seems to be technically more correct because we don't configure DNS|DHCP|TFTP (like setting up a DNS server or any DNS client settings), we configure Satellite to use an DNS|DHCP|TFTP service.